moped-editor/src/utils/validation.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Wrapper function for Yup.validateSync that returns formatted errors
+ * See https://github.com/jquense/yup?tab=readme-ov-file#schemavalidatesyncvalue-any-options-object-infertypeschema
+ * @param {object} value - Key-value pairs of the fields to validate
+ * @param {object} yupValidationSchema - The Yup validation schema to use
+ * @returns {object|null} - Returns an object with errors or null if no errors
+ */
+export const yupValidator = (value, yupValidationSchema) => {
```

Comment:
I added this helper in case we ever have a need for validation outside of React Hook Form. Not sure if that day will come or not.